### PR TITLE
fix(Select): fix custom colors

### PR DESCRIPTION
**Pull request by tjbo**
This PR fixes the custom colors which weren't being passed in properly for the recent Select Menu I added.

Just one thought on this; would be great to be able to be able to QA stuff (especially for me who is a bit of a noob at the RS thing).

I messed around a bit with BS-CONFIG, and a basic HTTP-SERVER, but thought I'd bring it up with you; to see what your thoughts are.

If you don't have time, I could implement something; let me know.

I found that RS build system to be a bit lacking; in terms of getting a fully functioning mounted package.

If it's build as a `commonjs` module, the require function is not included, and if it's build as `es6` we'd need to use something like an import map, which is probably not ideal:

```
<script>
{
  "imports": {
    "lodash": "/node_modules/lodash-es/lodash.js"
  }
}
</script>
```

Or I guess there are other more industrial options like webpack.

Any thoughts? 

It would allow myself and others to move a bit faster and steadier with QAing our code before opening a commit.

Instead of QAing after a release and having to make a commit like this. 

;)

### CODE REVIEW

File: src/Components/Chakra__Select.res [hunk 1]
```diff
@@ -628,6 +630,9 @@ let makeProps = (
     /**
       Component Props {Select}
      **/
+    ~errorBorderColor=?errorBorderColor->BorderColor.fromOption,
+    ~focusBorderColor=?focusBorderColor->BorderColor.fromOption,
+    ~iconColor=?iconColor->Color.fromOption,
     ~variant=?variant->Select.Variant.fromOption,
```
Comment (tjbo) [inline src/Components/Chakra__Select.res:636]: Not sure if there should be `IconColor`?

**Comment by ri7nz** [comment]
if you use `es6` you can see some project application like this https://github.com/ri7nz/rescript-pulo/blob/main/bsconfig.json .

But if you want to contribute changes for use `es6` in this project, you must configure `jest` configuration for `es6`. Before, I have been try `es6` but have some issue with `jest`. So, I'm skip that.

File: src/Components/Chakra__Select.res (hunk 1 above)
Comment (ri7nz) [reply src/Components/Chakra__Select.res:636]: No problem, this is okay.

**Pull request merged**